public/js/services/game.js

```diff
@@ -12,7 +12,7 @@ angular.module('mean.system')
     table: [],
     czar: null,
     playerMinLimit: 3,
-    playerMaxLimit: 6,
+    playerMaxLimit: 12,
```

Comment:
changing a constant value from two different sources is a potential for unpredictable behaviors which mar result in system failure. What if we can save this value in as an environmental variable or constant.js file so we could access it from one source.